lib/chat_api_web/controllers/setup_status_controller.ex

```diff
@@ -0,0 +1,73 @@
+defmodule ChatApiWeb.SetupStatusController do
+  use ChatApiWeb, :controller
+
+  alias ChatApi.{
+    Accounts,
+    BrowserSessions,
+    Github,
+    Google,
+    Mattermost,
+    SlackAuthorizations,
+    Twilio,
+    Users
+  }
+
+  action_fallback(ChatApiWeb.FallbackController)
+
+  @spec index(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def index(conn, _params) do
+    with current_user <- Pow.Plug.current_user(conn),
+         %{account_id: account_id} <- current_user do
+      account = Accounts.get_account!(account_id)
+
+      json(conn, %{
+        configured_profile: configured_profile?(current_user.id),
+        configured_storytime: configured_storytime?(account_id),
+        has_integrations: has_integrations?(account_id),
+        installed_chat_widget: installed_chat_widget?(account),
+        invited_teammates: invited_teammates?(account_id),
+        upgraded_subscription: upgraded_subscription?(account)
+      })
+    end
+  end
+
+  @spec configured_profile?(number()) :: boolean()
+  def configured_profile?(user_id) do
```

Comment:
i'm not always good about doing this myself, but this and the methods below can probably be marked with `defp` (for private)